source/kubernetes/index.html.md

```diff
@@ -1,31 +1,66 @@
 ---
-title: GOV.UK Kubernetes cluster documentation for developers
+title: GOV.UK Internal Developer Platform
 weight: 1
 layout: multipage_layout
 ---
 
-# GOV.UK Kubernetes cluster documentation for developers
+# GOV.UK Internal Developer Platform
 
-Use this documentation to find out how to:
+The GOV.UK Internal Developer Platform is a centralised platform where teams can build, deploy, and manage applications in a reliable and secure environment.
 
-- start using the GOV.UK Kubernetes clusters
-- manage your GOV.UK app on Kubernetes
+This documentation is designed for both platform users and internal team members, providing guides, tools, and best practices to onboard, manage, and troubleshoot your applications effectively.
 
-## Introduction
+## Who is this for?
 
-The GOV.UK website and content management system (known internally as the publishing platform) are hosted on [Kubernetes](https://kubernetes.io/) clusters on Amazon Web Services (AWS), using [Elastic Kubernetes Service](https://aws.amazon.com/eks/) (EKS).
+This documentation is for:
 
-Kubernetes is a widely-used, open-source [container orchestration system](https://cloud.google.com/discover/what-is-container-orchestration) for managing containerised workloads such as GOV.UK's.
+- Developers, engineers, and stakeholders deploying and managing applications on the platform.
 
-Kubernetes is designed for automation and emphasises [declarative configuration](https://github.com/kubernetes/design-proposals-archive/blob/main/architecture/declarative-application-management.md#declarative-configuration) and [immutable infrastructure](https://thenewstack.io/a-brief-look-at-immutable-infrastructure-and-why-it-is-such-a-quest/). It provides mechanisms for:
+- Platform engineers and administrators responsible for maintaining and extending the platform.
 
-- efficiently and automatically fitting workloads into available compute resources
-- resilient, "self-healing" deployments
-- service discovery and load balancing
-- automated rollouts and rollbacks
-- quotas and isolation of compute resources between workloads
-- authentication, authorisation and audit logging
-- secrets and configuration management
-- managing persistent storage
+## User Guide
+
+This section is specifically for application developers using the platform. The platform is designed to reduce cognitive load for developers by abstracting away infrastructure complexity and providing tools, guides, and automated workflows. 
+
+If there’s something missing, please either let us know and we’ll add a new article, or if you’re comfortable writing one yourself, PRs will be gratefully received.
+
+### Getting Started
+
+- [Gain access to a platform EKS Cluster](../../kubernetes/get-started/access-eks-cluster/index.html)
```

Comment:
Non-blocking or that important – I believe we don’t need to use absolute paths (which would save us some characters) but double check 🙈 
```suggestion
- [Gain access to a platform EKS Cluster](/kubernetes/get-started/access-eks-cluster/index.html)
```

Reply:
Absolutely, that's spot on. I have to use it for the local deployment to work, but I'll change it when it's moved to "Ready for Review".